File: 3.2/javascript/sum_of_range.js
// link in html and run in browser
var n = prompt("Set number: ");                 // pop-up prompt and get input
var arr = [...Array(parseInt(n)).keys()];       // parse input as an int
var sum = arr.reduce((a, b) => a + b, 0)        // make array of ints
alert("Nums: [" + arr + "]\n" + "Sum: " + sum)  // pop-up windows with Nums and Sum

File: 3.2/python/sum_of_range.py
i = input("Set number: ")           # print prompt and get input
n = int(i)                          # parse input as an int
arr = [n for n in range(n + 1)]     # make array of ints
print("Nums:", arr)                 # print Nums (as array)
print("Sum:", sum(arr))             # print Sum
